package.json

```diff
@@ -34,6 +34,7 @@
     "bignumber.js": "9.0.0",
     "bunyan": "^1.8.14",
     "config": "^3.3.1",
+    "json-bigint-native": "1.1.0",
```

Comment:
Using `bigint-native` to be able to do typecheck for `bigint`, even tho we are using the Bignumber library here as well.

Reply:
This has changed, we are using the BigNumber version now